app/src/main/java/com/kickstarter/viewmodels/PledgeFragmentViewModel.kt

```diff
@@ -880,7 +876,7 @@ interface PledgeFragmentViewModel {
 
             val createBackingSuccess = Observable.merge(this.stripeSetupResultSuccessful,
                     createBackingValues
-                            .map { it.backing().requiresAction() }
+                            .map { it.requiresAction() }
```

Comment:
Now looking again I don't think it's necessary to create a variable here either.